cfgov/unprocessed/apps/cfgov/unprocessed/apps/owning-a-home/package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
The creation of this file appears to be a mistake.

Reply:
No, it should be created. I'm not sure why it's empty though.

Reply:
Look at the path; it's in the wrong location.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3834